.coveragerc

```diff
@@ -210,6 +210,7 @@ omit =
     homeassistant/components/sensor/gtfs.py
     homeassistant/components/sensor/hp_ilo.py
     homeassistant/components/sensor/imap.py
+    homeassistant/components/sensor/knx.py
```

Comment:
You mean i should merge code of components/sensor/knx.py into /components/knx.py?